internal/storage/bucket_handle.go

```diff
@@ -235,6 +235,71 @@ func getProjectionValue(req gcs.Projection) storage.Projection {
 	return convertedProjection
 }
 
+func (b *bucketHandle) ListMinObjects(ctx context.Context, req *gcs.ListObjectsRequest)(listing *gcs.MinObjectListing, err error){
+    query := &storage.Query{
```

Comment:
all lines are not starting at same point. the spacing is not proper. please fix it.